client/src/pages/Dashboard.js

```diff
@@ -184,45 +169,100 @@ function UserInfo(props) {
 function ProjectCard(props) {
   const classes = useStyles();
   const projectInfo = props.project;
+  const fromNow = moment(projectInfo.deadline).fromNow();
+
   return (
     <Grid item xs={6}>
       <Card elevation={8}>
         <CardMedia
           className={classes.media}
           component="img"
-          src={projectInfo.photo}
+          src={
+            projectInfo.photos[0]
+              ? "https://plphotos.s3.us-east-2.amazonaws.com/" +
+                projectInfo.photos[0]
+              : ""
+          }
         ></CardMedia>
         <CardContent>
           <Typography className={classes.cardTitle} variant="h5" component="h4">
-            {projectInfo.name}
+            {projectInfo.title}
           </Typography>
           <Typography className={classes.cardInvested} display="inline">
-            ${projectInfo.currentInvested}
+            {projectInfo.current_invested}
           </Typography>
           <Typography color="textSecondary" display="inline">
-            {" / " + projectInfo.wantedInvestement}
+            {" / " + projectInfo.funding_goal}
           </Typography>
           <Typography variant="body2" color="textSecondary">
-            Equity exchange: {projectInfo.equity * 100}% |
-            {" " + projectInfo.timeLeft} days to go
+            Equity exchange: {projectInfo.equity * 100}% |{" " + fromNow}
           </Typography>
         </CardContent>
       </Card>
     </Grid>
   );
 }
 
-function UserDashboard(props) {
-  const classes = useStyles();
+function ErrorPage(props) {
   return (
-    <div className={classes.root}>
-      {/* Top Navbar */}
+    <div>
       <NavBar />
+      <Container align="center">{props.err}</Container>
+    </div>
+  );
+}
 
-      <Grid container>
+function UserDashboard(props) {
+  const classes = useStyles();
+  const [user, setUser] = useState();
+  const [err, setErr] = useState();
+  const [projects, setProjects] = useState({});
+
+  const location = useLocation();
+  const id = location.pathname.match(/profile\/(\d+)/);
+
+  useEffect(() => {
+    async function fetchData() {
+      try {
+        console.log("Fetching user data");
+        const userRes = await axios(`/api/v1/users/${id[1]}/profile`);
+        const projRes = await axios(`/api/v1/users/${id[1]}/projects`);
+        setUser(userRes.data);
```

Comment:
for asynchronous API calls, might be beneficial to split these into two other functions 
```
async function fetchUser() {
  const userRes = await axios(`/api/v1/users/${id[1]}/profile`);
  setUser(userRes.data);
}

async function fetchProject() {
  const projRes = await axios(`/api/v1/users/${id[1]}/projects`);
  setProjects(projRes.data);
}
```

```suggestion
       fetchUser();
       fetchProject(); // this way both of these are getting fired off at the same time, not dependant on each other
```

Reply:
That's a really good idea